src/frontend/src/pages/HomePage/components/WorkPackagesSelectionView.tsx

```diff
@@ -0,0 +1,130 @@
+import { RoleEnum, wbsPipe, WorkPackage } from 'shared';
+import { Box, Grid, useTheme } from '@mui/material';
+import {
+  getInProgressWorkPackages,
+  getOverdueWorkPackages,
+  getUpcomingWorkPackages
+} from '../../../utils/work-package.utils';
+import { useCurrentUser } from '../../../hooks/users.hooks';
+import PageBlock from '../../../layouts/PageBlock';
+import WorkPackageCard from './WorkPackageCard';
+import WorkPackageSelect from './WorkPackageSelect';
+import React, { useState } from 'react';
+import EmptyPageBlockDisplay from './EmptyPageBlockDisplay';
+import CheckCircleOutlineOutlinedIcon from '@mui/icons-material/CheckCircleOutlineOutlined';
+
+const NoWorkPackages: React.FC = () => {
+  return (
+    <Box
+      sx={{
+        width: '100%',
+        height: '40vh',
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'center',
+        alignItems: 'center'
+      }}
+    >
+      <EmptyPageBlockDisplay
+        icon={<CheckCircleOutlineOutlinedIcon sx={{ fontSize: 70 }} />}
+        heading={`You're all caught up!`}
```

Comment:
Not a big deal, but this doesn't make a ton of sense regarding upcoming work packages. Maybe change to a more generic message like "You're all set!"